dockerfiles/ci/centos/7/php-8.3/configure.sh

```diff
@@ -0,0 +1,47 @@
+if [ -z "${PHP_SRC_DIR}" ]; then
+    echo "Please set PHP_SRC_DIR"
+    exit 1
+fi
+
+# needed for CentOS7 vs PHP 8.3, otherwise you get a:
+# ld: dynamic STT_GNU_IFUNC symbol `mb_wchar_to_utf16le' with pointer equality in `ext/mbstring/libmbfl/filters/mbfilter_utf16.o' can not be used when making an executable; recompile with -fPIE and relink with -pie
+# see also https://github.com/php/php-src/issues/11603
+export LDFLAGS=-pie
+export CFLAGS=-fPIE
```

Comment:
@bwoebi can you help me with this? I am in the impression, that this should ne be necessary, but it is for CentOS 7 on `x86_64` for PHP 8.3. I do not need those flags for `arm64` or PHP 8.2 or for PHP 8.3 on Ubuntu / `x86_64`.
Can you help me understand whats going on?

Reply:
Thanks @bwoebi for the quick discussion, seems like this is related to CentOS 7 working with an outdated gcc, so we'll leave this there and guard it to only be set on `x86_64`.